hazelcast/src/main/java/com/hazelcast/cache/impl/journal/RingbufferCacheEventJournalImpl.java

```diff
@@ -93,7 +96,14 @@ public long oldestSequence(ObjectNamespace namespace, int partitionId) {
 
     @Override
     public void destroy(ObjectNamespace namespace, int partitionId) {
-        getRingbufferService().destroyContainer(partitionId, namespace);
+        final RingbufferService service;
+        try {
+            service = getRingbufferService();
+        } catch (Exception e) {
+            logger.fine("Could not destroy event journal " + namespace, e);
+            return;
+        }
+        service.destroyContainer(partitionId, namespace);
```

Comment:
You're right, the message could be clearer in this sense. I've purposefully surrounded only the `getService` call - I would allow any exception from `destroyContainer` to be bubbled up the call stack - just as other services usually don't catch exceptions in `destroy`. WDYT?